bin/concert

```diff
@@ -23,6 +23,19 @@ from concert.ext.cmd import plugins
 logging.addLevelName(concert.config.PERFDEBUG, 'PERFDEBUG')
 logging.addLevelName(concert.config.AIODEBUG, 'AIODEBUG')
 LOG = logging.getLogger(__name__)
+STARTUP_FMT = """
+try:
+    from {0} import *
+except Exception as e:
+    import os
+    import signal
+    import sys
+    import traceback
+    print("\\nAn error occured while starting session `{0}':")
+    print("-------------------------------------------" + "-" * len('{0}'))
+    print(traceback.format_exc(), file=sys.stderr)
+    os.kill(os.getpid(), signal.SIGTERM)
+"""
```

Comment:
@MarcusZuber do we want to remove the lock file if this happens?

Reply:
You wanted to keep it, so one needs to think if something breaks.
(I hated it while I was debugging sessions)

Reply:
That's why I asked. Hm, I don't know. I mean technically if there is an error the session is not even started, so the lock file does not make sense. I will try to add the removal, OK?